commons/all/src/main/java/org/infinispan/commons/configuration/io/AbstractConfigurationWriter.java

```diff
@@ -95,23 +127,26 @@ protected void outdent() {
    }
 
    @Override
-   public void close() throws Exception {
-      writer.close();
+   public void close() {
+      Util.close(writer);
    }
 
    public static class Tag {
       final String name;
       final boolean repeating;
-      final boolean explicit;
+      final boolean explicitOuter;
+      final boolean explicitInner;
+      int children;
```

Comment:
`incrementChildren()` unused. This field can be removed too.

Reply:
Removed